packages/ckeditor5-dev-utils/lib/bundler/getlicensebanner.js

```diff
@@ -10,7 +10,7 @@ module.exports = function getLicenseBanner() {
 
 	/* eslint-disable indent */
 	return (
-`/**
+`/*!
```

Comment:
I changed comment, so it differs now a little bit from the original comments used in the code. It's the result of https://github.com/webpack/webpack/issues/6630#issuecomment-400990552.

Reply:
Can we use `/*` (with one `*`)? This `!` looks like a typo.

Reply:
Or something like:
```
/*-------------------------*
 *                         *
 *-------------------------*/
```
Anything what does not look like a typo.

Reply:
I'm ok with

```
/*-------------------------*
 *                         *
 *-------------------------*/
```

It just needs to differ from original license headers.